controllers/AuthController.js

```diff
@@ -24,7 +22,7 @@ class AuthController {
 
     const genSalt = bcrypt.genSaltSync(8);
     const hashPassword = bcrypt.hashSync(password, genSalt);
-    const role = 'user';
+    const roleName = 'user';
```

Comment:
Sunny, is this necessary after you've set the default value to `user` in the model?